features/raadpleeg-verblijfplaats-met-periode/gba/overzicht-gba.feature

```diff
@@ -0,0 +1,330 @@
+
+  Abstract Scenario: datumTot van de gevraagde periode is gelijk aan de eerste dag van de deels onbekende datum aanvang adreshouding (geen vorige en geen volgende verblijfplaats)
+    Gegeven de persoon met burgerservicenummer '000000024' is ingeschreven op adres 'A1' met de volgende gegevens
+    | gemeente van inschrijving (09.10) | datum aanvang adreshouding (10.30) |
+    | 0800                              | <datum aanvang adreshouding>       |
+    Als gba verblijfplaatsen wordt gezocht met de volgende parameters
+    | naam                | waarde              |
+    | type                | RaadpleegMetPeriode |
+    | burgerservicenummer | 000000024           |
+    | datumVan            | 2007-01-01          |
+    | datumTot            | <datum tot>         |
+    Dan heeft de response verblijfplaatsen met de volgende gegevens
+    | adresseerbaarObjectIdentificatie | straat | datumAanvangAdreshouding     |
+    | 0800010000000001                 | Laan   | <datum aanvang adreshouding> |
+
+    Voorbeelden:
+    | datum aanvang adreshouding | datum tot  |
+    | 20100800                   | 2010-08-01 |
+    | 20100000                   | 2010-01-01 |
+
+  Abstract Scenario: <scenario> (wel vorige en geen volgende verblijfplaats)
+    Gegeven de persoon met burgerservicenummer '000000024' is ingeschreven op adres 'A1' met de volgende gegevens
+    | gemeente van inschrijving (09.10) | datum aanvang adreshouding (10.30) |
+    | 0800                              | 20100810                           |
+    En de persoon is vervolgens ingeschreven op adres 'A2' met de volgende gegevens
+    | gemeente van inschrijving (09.10) | datum aanvang adreshouding (10.30) |
+    | 0800                              | 20160000                           |
+    Als gba verblijfplaatsen wordt gezocht met de volgende parameters
+    | naam                | waarde              |
+    | type                | RaadpleegMetPeriode |
+    | burgerservicenummer | 000000024           |
+    | datumVan            | <datum van>         |
+    | datumTot            | <datum tot>         |
+    Dan heeft de response verblijfplaatsen met de volgende gegevens
+    | adresseerbaarObjectIdentificatie | straat | datumAanvangAdreshouding     |
+    | 0800010000000001                 | Laan   | <datum aanvang adreshouding> |
+
+    Voorbeelden:
+    | datum van  | datum tot  | scenario                                                                                               |
+    | 2016-02-02 | 2016-12-02 | de gevraagde periode ligt in de onzekerheidsperiode van de adreshouding                                |
+    | 2016-07-01 | 2017-07-01 | datumVan ligt in de onzekerheidsperiode en datumTot ligt na de onzekerheidsperiode van de adreshouding |
+    | 2017-01-01 | 2020-01-01 | de gevraagde periode ligt na de onzekerheidsperiode van de adreshouding                                |
```

Comment:
OK. Dit maakt allemaal niets uit. Je krijgt het complete verblijfplaats voorkomen als deze (deels) misschien of (deels) zeker in de gevraagde periode ligt.

Reply:
ja, omdat de  eerste dag van de onzekerheidsperiode als datum aanvang wordt gehanteerd. Dan kan de rule voor bekend datum aanvang worden gehanteerd om te bepalen voor het verblijfplaats voorkomen moet worden geleverd